reference/configuration/monolog.rst

```diff
@@ -74,6 +74,11 @@ Full Default Configuration
                         id:                   ~ # Required (when the email_prototype is used)
                         method:               ~
                     formatter:            ~
+            # Setting this parameter to false forces the logger to reduce
+            # the precision in the datetime field of the log messages from
+            # microsecond to second. Avoiding a call to the microtime(true)
+            # function and the subsequent parsing.
+            use_microseconds:             true
```

Comment:
I agree! This is a little long :). How about:

```yml
Set to false to use seconds (instead of microseconds) in the logs (gives a small performance boost)
```